launch/launcher_test.go

```diff
@@ -256,7 +261,7 @@ func testLauncher(t *testing.T, when spec.G, it spec.S) {
 					{Type: "start", Command: "./start"},
 				}
 				launcher.Buildpacks = []launch.Buildpack{{ID: "bp.1"}, {ID: "bp.2"}}
-				launcher.Exec = syscallExecWithStdout(t, tmpDir)
+				launcher.Exec = testsyscall.SyscallExecWithStdout(t, tmpDir)
```

Comment:
Will do. Moving into `testhelpers` package, as there were compilation issues with moving them into `launcher_test` (I think because the file names don't end in `_test.go`).